### PR TITLE
Add task in allinone

### PR DESCRIPTION
**Pull request by hxsz1997**
Add task (summarize/QA/continuation) in all-in-one

### CODE REVIEW

File: python/llm/dev/benchmark/all-in-one/run.py [hunk 1]
```diff
@@ -463,28 +463,47 @@ def run_transformer_int4_gpu(repo_id,
...
+                    test_length = test_length * 2
+                # For the sequence length not in [32, 256, 1024, 2048, 8192], it will be truncated from 8192.txt.
+                test_length = min(test_length, 8192)
+                input_str = open(f"prompt/continuation/{test_length}.txt", 'r').read()
                 # As different tokenizer has different encodings,
                 # slice the input_ids to ensure the prompt length is required length.
                 input_ids = tokenizer.encode(input_str, return_tensors="pt")
                 input_ids = input_ids[:, :in_len]
+            elif conf['task'] == 'summarize':
+                if in_len == 512:
```
Comment (cyita) [inline python/llm/dev/benchmark/all-in-one/run.py:481]: Maybe add a map & deal with the in_len not in [512, 1024, 2048] case.

Comment (cyita) [reply python/llm/dev/benchmark/all-in-one/run.py:481]: Add the short prompt cnn-64.

Comment (hxsz1997) [reply python/llm/dev/benchmark/all-in-one/run.py:481]: Thanks. I have added these cases.

File: python/llm/dev/benchmark/all-in-one/run.py [hunk 2]
```diff
@@ -463,28 +463,53 @@ def run_transformer_int4_gpu(repo_id,
...
+                    input_str = open(f"prompt/summarize/cnn_239.txt", 'r').read()
+                elif in_len == 1024:
+                    input_str = open(f"prompt/summarize/cnn_615.txt", 'r').read()
```
Comment (cyita) [inline python/llm/dev/benchmark/all-in-one/run.py:484]: Make the summarize & QA prompt preprocessing as a util function.

Comment (hxsz1997) [reply python/llm/dev/benchmark/all-in-one/run.py]: Sure. I have fixed it.

**Review by cyita — APPROVED** [review]
LGTM

**Pull request merged**